src/shared.js

```diff
@@ -43,10 +45,11 @@ export function parseHeader (str = '') {
  * Attempts to safely parse 'nplurals" value from "Plural-Forms" header
  *
  * @param {Object} [headers = {}] An object with parsed headers
+ * @param fallback {Number} Fallback value
  * @returns {number} Parsed result
  */
-export function parseNPluralFromHeadersSafely (headers = {}, fallback = 1) {
-  const pluralForms = headers[PLURAL_FORMS];
+export function parseNPluralFromHeadersSafely (headers, fallback = 1) {
+  const pluralForms = headers ? headers[PLURAL_FORMS] : false;
 
   if (!pluralForms) {
     return fallback;
```

Comment:
same issue with null checks here... if the headers is an object the `if (!pluralForms) {` will never be fired